core/cmd/remote_client.go

```diff
@@ -548,3 +555,39 @@ func parseResponse(resp *http.Response) ([]byte, error) {
 	}
 	return b, err
 }
+
+func (cli *Client) checkRemoteBuildCompatibility(lggr logger.Logger, c *clipkg.Context) error {
+	resp, err := cli.HTTP.Get("/v2/build_info")
+	if err != nil {
+		lggr.Warn("Remote node version is unknown. CLI may behave in unexpected ways.")
+		return nil
+	}
+	b, err := parseResponse(resp)
+	if err != nil {
+		lggr.Warn("Remote node version is unknown. CLI may behave in unexpected ways.")
+		return nil
+	}
+
+	var remote_build_info map[string]string
```

Comment:
The logic looks good now, but can we switch to `mixedCaps` for all these variables?

Reply:
woops, still getting used to go conventions